google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/BigtableDataClient.java

```diff
@@ -333,6 +334,33 @@ public void readRowsAsync(Query query, ResponseObserver<Row> observer) {
     return stub.mutateRowCallable();
   }
 
+  /**
+   * Mutates multiple rows in a batch. Each individual row is mutated atomically as in MutateRow,
+   * but the entire batch is not executed atomically.
+   *
+   * <p>Sample code:
+   *
+   * <pre>{@code
+   * InstanceName instanceName = InstanceName.of("[PROJECT]", "[INSTANCE]");
+   * try (BigtableClient bigtableClient = BigtableClient.create(instanceName)) {
+   *   try (BulkMutations mutations = bigtableClient.newBulkMutations()) {
+   *     for(String someValue in someCollection) {
+   *       RowMutation mutation = RowMutation.create("[TABLE]", "[ROW KEY]")
+   *         .setCell("[FAMILY NAME]", "[QUALIFIER]", "[VALUE]");
+   *
+   *       mutations.send(mutation);
+   *     }
+   *   } catch (BulkMutationFailure failure) {
+   *     // Handle error
+   *   }
+   *   // After `mutations` are closed, all mutations have been applied
```

Comment:
I'm not sure if this should be `is`. After all, `mutations` here is one variable, not actually many mutations.

Reply:
I think you are right